server/controllers/wazuh-hosts.js

```diff
@@ -117,4 +129,32 @@ export class WazuhHostsCtrl {
       );
     }
   }
+
+  async checkAPIUserAllowRunAs(apiId){
```

Comment:
this looks the same as the function into wazuh-api.js, if this is true you could put it into a separate file and use it from both classes.

Reply:
Now, it is using the refactorized function instead repeat the same code.